lips/lip-6.md

```diff
@@ -0,0 +1,79 @@
+---
+lip: 6
+title: Libra Validator Configuration Management
+author: Valeria Nikolaenko, David Wolinsky, Sam Blackshear, David Dill
+status: Draft
+type: Informational
+created: 07/30/2020
+---
+# Summary
+This LIP describes the conceptual model and implementation of validator configuration management on the Libra blockchain.
+
+# Abstract / Motivation
+The Libra Payment Network (“LPN”) is a blockchain-backed payment platform and an infrastructure for stablecoins that supports a regulated global digital payment network. Both the blockchain and the stablecoin financial instruments are controlled by Libra Networks, Ltd. (“Libra Networks” or “LN”) a subsidiary of the Libra Association  (the “Association” or “LA”).  LPN operates under a payment system license from FINMA (the “FINMA License”), and LN is responsible for ensuring that LPN operates in compliance with the FINMA License.
+
+The Libra Association is an independent membership organization that is responsible for the governance of the LPN and the development of the Libra project. The Association is governed by the Council, which is comprised of one representative per Association member. The Association Council manages the technology roadmap of the platform and oversees and maintains the Libra Improvement Proposal and Libra Upgrade Process, often through subcommittees of the Council such as the Technical Steering Committee (the “TSC”) and the Association Board. 
+
+Each member that participates in maintaining the blockchain by operating a Validator Node or designating a party to operate a Validator Node is referred to as a Validator Owner. The Validator Set defines the current set of Validator Nodes that run consensus protocol. Validator Nodes that are currently part of the Validator Set process transactions and interact with each other to reach consensus on the state of the blockchain. The Validator Set may change from time to time and not all members may have Validator Nodes in the Validator Set at any given time. Reasons for changing the Validator Set include joining or departure of members from the Association and actions needed for compliance reasons or to comply with LN’s obligations under its payment system license from FINMA to guarantee the secure and performant operation of the platform.
+
+# Terminology
+* *Validator Node* ("node") is a replica that runs the Libra Protocol to maintain a database of programmable resources. Validator Node processes transactions and interacts with other Validator Nodes to reach consensus on the ordering of transactions and the resulting state of the database after transactions' executions. 
```

Comment:
Should we say that we refer to "validator nodes" as "Validators" in the rest of the doc?

Reply:
Not sure actually, this might become very confusing. I tried to make sure everywhere I use full names ("Validator Node" or "Validator Operator" or "Validator Owner"), though this is a little cumbersome indeed.